isatty_linux_ppc64.go

```diff
@@ -0,0 +1,18 @@
+// +build linux,ppc64
```

Comment:
ppc64le can't see this file, the filename should not be _ppc64, maybe _ppc64x.go is fine. And the tag should add linux,ppc64le too.

Reply:
Sorry, I can't reproduce it because I don't have ppc64 or ppc64le. Do this occur only on ppc64le?

Reply:
Hi, ppc64 and ppc64le both have the problem.

Reply:
Oops. will fix.